Creation of documentation/implementation guide for Worklist DataGrid (#1826)
cerner/terra-framework: packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/PinnedColumns.5.doc.mdx

import PinnedColumns from './PinnedColumns?dev-site-example';

# Pinned Columns

### Description

> The [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) defines two types of columns: Pinned and Overflow. The pinned columns are aligned to the left of the Worklist DataGrid and cannot be scrolled. The overflow section is aligned to the right of pinned columns and scrolls horizontally to reveal each overflow column. This example demonstrates a Worklist DataGrid that supports both pinned and overflow columns.

Reviewer: ```suggestion
The [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) defines two types of columns: Pinned and Overflow columns.
Pinned columns will stay stickied to the left of the WorklistDataGrid when scrolling horizontally.
The overflow column section can be scrolled horizontally.
This example demonstrates a WorklistDataGrid that supports both pinned and overflow columns.
```

Author: Changed in previous commit
